src/mobster/oci/cosign.py

```diff
@@ -91,6 +146,27 @@ async def fetch_latest_provenance(self, image: Image) -> Provenance02:
 
         return sorted(provenances, key=lambda x: x.build_finished_on, reverse=True)[0]
 
+    async def fetch_attested_sbom(self, image: Image, sbom_format: SBOMFormat) -> SBOM:
+        """
+        Fetch attested SBOM.
+        Args:
+            image: The image this attestation (and the SBOM) belongs to
+            sbom_format: The expected SBOM format. This function only cares if the type
+                is SPDX or CDX, specific version does not matter
+
+        Returns:
+            The initialized SBOM object
+        """
+        attestation_type = self.__get_cosign_attestation_type(sbom_format)
+        last_attestation = (
+            await self._perform_cosign_verify_attestation(image, attestation_type)
+        )[-1]
```

Comment:
I think this should verify if the attestation exists, otherwise, I think it can raise IndexError.